package-overrides/npm/angular2@2.0.0-beta.8.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "zone.js": "npm:zone.js@0.6.2",
```

Comment:
And why are es6-promise and es6-shim not included? See package.json: https://github.com/angular/angular/blob/master/modules/angular2/package.json